germansentiment/sentimentmodel.py

```diff
@@ -21,7 +21,8 @@ def __init__(self, model_name: str = "oliverguhr/german-sentiment-bert"):
     def predict_sentiment(self, texts: List[str])-> List[str]:
         texts = [self.clean_text(text) for text in texts]
         # Add special tokens takes care of adding [CLS], [SEP], <s>... tokens in the right way for each model.
-        input_ids = self.tokenizer.batch_encode_plus(texts,padding=True, add_special_tokens=True)
+        # limit number of tokens to model's limitations (512)
+        input_ids = self.tokenizer.batch_encode_plus(texts,padding=True, add_special_tokens=True,truncation=True,max_length=512)
```

Comment:
We can use tokenizer.nmodel_max_length to determine the models max length. This way it will work for models like distillbert too.